botbuilder-instumentation/Instumentation/BotFrameworkInstrumentation.cs

```diff
@@ -131,6 +132,19 @@ public void TrackQnaEvent(IActivity activity, string userQuery, string kbQuestio
             }
         }
 
+        public void TrackGoalTriggeredEvent(IActivity activity, string goalName,
+            IDictionary<string, string> goalTriggeredEventProperties = null, string eventName = TelemetryEventTypes.GoalTriggeredEvent)
```

Comment:
Thanks for this. I will update as soon as I get a chance and submit a new pull request with updated code.